docs/bis/ec/ec-custom-attributes.md

```diff
@@ -43,4 +43,17 @@ Custom attributes allow any custom metadata to be applied because they are defin
     </ECCustomAttributeClass>
 ```
 
+It is also possible to query custom attributes:
```

Comment:
this doesn't belong here ... this is just EC reference we don't have anything talking about ECDb.  Maybe we need a meta query section inn the ECSql Reference.  In there we need some examples that include getting the values of custom attributes

Reply:
done